sydent/sydent.py

```diff
@@ -73,6 +73,7 @@ class Sydent:
         'log.path': '',
         'pidfile.path': 'sydent.pid',
         'ed25519.signingkey': '',
+        'bodyTemplate': 'Your code is {token}',
```

Comment:
You'd think so, but in fact the defaults are a separate section, oddly, so you don't add the prefixes. Making a PR to change the rest.